request.js

```diff
@@ -116,15 +116,15 @@
         return callbacks;
     };
 
-    exports['get'] = function (url, query) {
+    exports.get = function (url, query) {
```

Comment:
No, its just to keep the "visual pattern". No problem with me; if @maxigimenez agrees, go for it.